src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/nowhitespaceafter/InputNoWhitespaceAfterFormerNpe.java

```diff
@@ -0,0 +1,8 @@
+package com.puppycrawl.tools.checkstyle.checks.whitespace.nowhitespaceafter;
+
+public class InputNoWhitespaceAfterFormerNpe
```

Comment:
@rnveach 
Could you please review it again. 
At last push, I faced with the problem that TeamCity build and wercker pipeline are not passing. TeamCity arguing on files that I haven't modified and also I haven't access to wercker.
Thanks